app/javascript/components/patient/household/utils/IconMinor.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
So I'm not sure this is the best place to put this file given it's not household related, however there doesn't seem to be another great place to put it unless we move both Icon/Badge components into a general patient utils folder.  @holmesie thoughts?

Reply:
So, a `utils` one directory back (i.e. `app/javascript/components/patient/utils/IconMinor.js`)? I think that's a good idea, and a better fit for what this component does.

Reply:
yes that's exactly what I was thinking (would be a new file).  just wasn't sure if it made sense to only have one component in there or name it something like `Icons` and move the HoH badge too

Reply:
that way we don't have tons of utils folders (we already have a couple)

Reply:
Yeah that seems reasonable to me!